assets/js/docs-feedback.js

```diff
@@ -0,0 +1,72 @@
+$(function(){
+    // Check that the analytics track function is available.
+    var analyticsAvailable = window.analytics && window.analytics.track &&
+                             (typeof window.analytics.track === "function");
+
+    if (analyticsAvailable) {
+        // Show the docs feedback container.
+        $("#docsFeedbackContainer").removeClass("hidden");
+
+        // For each of the button ids attach a click handler that shows
+        // the additional comment section.
+        ["#docsFeedbackYes", "#docsFeedbackNo"].forEach(function(key) {
+            var answer = key === "#docsFeedbackYes" ? "Yes" : "No";
+            $(key).on("click", function() {
```

Comment:
Noticed after submitting that when I clicked a thumb again on the same page, the fields were still there. We should probably clear the fields on successive clicks to open this form (and probably clear them on successful submission as well) -- not a huge deal of course, since users will be navigating around, but still, keeps things just a smidge tidier that way.